### DISCUSSION TITLE
How to disable burn debug statements from being copied to the log file?

### DISCUSSION THREAD

**Discussion by nagacharan-tangirala** [tracel-ai/burn#2567] [Q&A] [answered]
I am using burn in an application that generates a log file. However, all the burn debug statements are also routed to this file. How do you think I could prevent this?

**Comment by laggui** [comment, marked as answer]
As far as I know, this depends on what you are using to log stuff in your crate/application. Perhaps this [SO post](https://stackoverflow.com/questions/73247589/how-to-turn-off-tracing-events-emitted-by-other-crates) is relevant to filter logs with `tracing`.
